### PR TITLE
Added market hours and indices

### PR DESCRIPTION
**Pull request by justinpolygon**
Added market hours and indices so that these match the spec.

```
MarketStatusDTO(
  Companion = Companion(
  )
  market = "open"
  serverTime = "2023-07-13T13:36:42-04:00"
  afterHours = false
  earlyHours = false
  exchanges = ExchangesStatusDTO(
    Companion = Companion(
    )
    nyse = "open"
    nasdaq = "open"
    otc = "open"
  )
  indicesGroups = IndicesGroupsStatusDTO(
    Companion = Companion(
    )
    s_and_p = "open"
    societe_generale = "open"
    msci = "open"
    ftse_russell = "open"
    mstar = "open"
    mstarc = "open"
    cccy = "open"
    nasdaq = "open"
    dow_jones = "open"
  )
  currencies = CurrenciesStatusDTO(
    Companion = Companion(
    )
    fx = "open"
    crypto = "open"
  )
)
```

### CODE REVIEW

File: src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/MarketStatus.kt [hunk 1]
```diff
@@ -26,8 +29,21 @@ data class ExchangesStatusDTO(
...
+    val s_and_p: String? = null,
+    val societe_generale: String? = null,
+    val msci: String? = null,
+    val ftse_russell: String? = null,
+    val mstar: String? = null,
+    val mstarc: String? = null,
+    val cccy: String? = null,
+    val nasdaq: String? = null,
+    val dow_jones: String? = null
+)
```
Comment (mmoghaddam385) [inline src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/MarketStatus.kt:43]: Fields in here should be camelCase w/ the `@SerialName` annotation when necessary for consistency.

Comment (justinpolygon) [reply src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/MarketStatus.kt]: Fixed.

Comment (mmoghaddam385) [reply src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/MarketStatus.kt]: sorry I meant only the fields where the `@SerialName` has to be snake_case need the annotation, single word ones like `nasdaq` don't, but it's fine to keep them.

**Review by mmoghaddam385 — APPROVED** [review]

File: src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/MarketStatus.kt (hunk 1 above)
Comment (justinpolygon) [reply src/main/kotlin/io/polygon/kotlin/sdk/rest/reference/MarketStatus.kt]: Yeah, that makes sense. My mistake. Fixed.

**Pull request merged**